DSharpPlus/Entities/DiscordGuild.cs

```diff
@@ -421,30 +422,31 @@ public Task<DiscordChannel> CreateChannelCategoryAsync(string name, IEnumerable<
         /// <param name="reason">Reason for audit logs.</param>
         /// <returns>The newly-created channel.</returns>
         public Task<DiscordChannel> CreateVoiceChannelAsync(string name, DiscordChannel parent = null, int? bitrate = null, int? user_limit = null, IEnumerable<DiscordOverwriteBuilder> overwrites = null, string reason = null)
-            => this.CreateChannelAsync(name, ChannelType.Voice, parent, bitrate, user_limit, overwrites, null, Optional.FromNoValue<int?>(), reason);
+            => this.CreateChannelAsync(name, ChannelType.Voice, parent, null, bitrate, user_limit, overwrites, null, Optional.FromNoValue<int?>(), reason);
```

Comment:
After changing to optional it will need to be changed from `null` to `Optional.FromNoValue<string>()`.